pkgs/common-updater/scripts/mark-broken

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+set -e
+
+scriptName=mark-broken # do not use the .wrapped name
+
+die() {
+    echo "$scriptName: error: $1" >&2
+    exit 1
+}
+
+usage() {
+    echo "Usage: $scriptName <attr> [--new-value=<new-value>]"
+}
+
+args=()
+
+for arg in "$@"; do
+    case $arg in
+        --new-value=*)
+            newValue="${arg#*=}"
+        ;;
+        --help)
+            usage
+            exit 0
+        ;;
+        --*)
+            echo "$scriptName: Unknown argument: $arg"
+            usage
+            exit 1
+        ;;
+        *)
+            args["${#args[*]}"]=$arg
+        ;;
+    esac
+done
+
+attr=${args[0]}
+
+if (( "${#args[*]}" < 1 )); then
+    echo "$scriptName: Too few arguments"
+    usage
+    exit 1
+fi
+
+if (( "${#args[*]}" > 1 )); then
+    echo "$scriptName: Too many arguments"
+    usage
+    exit 1
+fi
+
+if [ -z $newValue ]; then
+  newValue="true"
+fi
+
+nixFile=$(nix-instantiate --eval --json -E "with import ./. {}; (builtins.unsafeGetAttrPos \"description\" $attr.meta).file" | jq -r .)
+if [[ ! -f "$nixFile" ]]; then
+    die "Couldn't evaluate 'builtins.unsafeGetAttrPos \"meta\" $attr' to locate the .nix file!"
```

Comment:
Hope you're not sick of me yet... ;)

```suggestion
    die "Couldn't evaluate 'builtins.unsafeGetAttrPos \"description\" $attr' to locate the .nix file!"
```

Somewhat unfortunate that `meta` points to `pkgs/stdenv/generic/make-derivation.nix` for the rare cases (if any) where a package doesn't have a `description`, but what can ya do...

Reply:
Yeah that

Reply:
Yeah I thought about what meta attribute is set for almost all packages. If there is a better way to find the file containing the meta section for a nixpkgs attribute, it would be great to know it.
Otherwise we can try to improve the checks to detect if it found the wrong file, for example a specific check for make-derivation.nix.

Reply:
I committed something similar to your suggested change, but I will still leave this conversation open, because I think there should be more improvements.

Reply:
There is `<pkg>.meta.position`, but that doesn't work for things like [`cargo-download`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/package-management/cargo-download/default.nix) which returns `pkgs/build-support/rust/build-rust-crate/default.nix:224` (and includes the line number, which would need to be stripped out).

Reply:
Something to keep in mind with your new wording here: trying to evaluate `builtins.unsafeGetAttrPos \"meta.description\" claws` (as displayed) returns `null`, while `builtins.unsafeGetAttrPos \"description\" claws.meta` (which is what is actually evaluated) returns the desired information.